ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ipv6-static-routes.j2

```diff
@@ -1,15 +1,34 @@
-{# static routes#}
-{% if ipv6_static_routes is defined and ipv6_static_routes is not none %}
+{# eos - IPV6 static routes#}
+{% if ipv6_static_routes is arista.avd.defined %}
 !
-{%     for ipv6_static_route in ipv6_static_routes %}
-ipv6 route
-{%- if ipv6_static_route['vrf'] is defined and ipv6_static_route['vrf'] != 'default' and ipv6_static_route['vrf'] is not none %} vrf {{ ipv6_static_route['vrf'] }}{% endif %} {{ ipv6_static_route['destination_address_prefix'] }}
-{%- if ipv6_static_route['interface'] is defined and ipv6_static_route['interface'] is not none and ipv6_static_route['interface'] | capitalize != 'Null0' %} {{ ipv6_static_route['interface'] | capitalize }}{% elif ipv6_static_route['interface'] is defined and ipv6_static_route['interface'] is not none and ipv6_static_route['interface'] | capitalize == 'Null0' %} {{ ipv6_static_route['interface'] | capitalize }}{%- endif -%}
-{%- if ipv6_static_route['gateway'] is defined and ipv6_static_route['gateway'] is not none and (static_route['interface'] is not defined or ipv6_static_route['interface'] | capitalize != 'Null0') %} {{ ipv6_static_route['gateway'] }}{% endif -%}
-{%- if ipv6_static_route['distance'] is defined and ipv6_static_route['distance'] is not none %} {{ ipv6_static_route['distance'] }}{% endif -%}
-{%- if ipv6_static_route['tag'] is defined and ipv6_static_route['tag'] is not none %} tag {{ ipv6_static_route['tag'] }}{% endif -%}
-{%- if ipv6_static_route['name'] is defined and ipv6_static_route['name'] is not none %} name {{ ipv6_static_route['name'] }}{% endif %}
-{%- if ipv6_static_route['metric'] is defined and ipv6_static_route['metric'] is not none %} metric {{ ipv6_static_route['metric'] }}{% endif %}
-
-{%          endfor %}
-{% endif %}
\ No newline at end of file
+{%      for ipv6_static_route in ipv6_static_routes %}
+{%          set ipv6_static_route_cli = "ipv6 route" %}
+{%          if ipv6_static_route.vrf is arista.avd.defined %}
+{%              if ipv6_static_route.vrf != 'default' %}
+{%                  set ipv6_static_route_cli = ipv6_static_route_cli ~ " vrf " ~ ipv6_static_route.vrf %}
+{%              endif %}
+{%          endif %}
+{%          if ipv6_static_route.destination_address_prefix is arista.avd.defined %}
+{%              set ipv6_static_route_cli = ipv6_static_route_cli ~ " " ~ ipv6_static_route.destination_address_prefix %}
+{%          endif %}
+{%          if ipv6_static_route.interface is arista.avd.defined %}
+{%              set ipv6_static_route_cli = ipv6_static_route_cli ~ " " ~ ipv6_static_route.interface | capitalize %}
+{%          endif %}
+{%          if ipv6_static_route.gateway is arista.avd.defined and (ipv6_static_route.interface is not arista.avd.defined or ipv6_static_route.interface | capitalize != 'Null0')%}{#Checking if Gateway is defined and Interface is not defined or Interface is null0. In this case, no gateway is needed.#}
```

Comment:
Move comment above if statement

Reply:
The comment was introducing a newline. Probably we can delete it.

Reply:
Yes you can also delete it.